plugin-mainmenu/lxqtmainmenu.cpp

```diff
@@ -653,6 +653,10 @@ bool LXQtMainMenu::eventFilter(QObject *obj, QEvent *event)
     {
         if(event->type() == QEvent::KeyRelease)
         {
+            if (mShortcutSeq.isEmpty())
```

Comment:
I'm afraid, with this the "first letter" navigation in menus feature is forbidden, when the shortcut is missing/failed.

Reply:
Thanks! You're completely right. I didn't pay attention to what came below those lines, in the case of Main Menu; will fix it soon.